drivers/flash/flash_handlers.c

```diff
@@ -65,7 +65,7 @@ static inline int z_vrfy_flash_get_page_info_by_idx(struct device *dev,
 }
 #include <syscalls/flash_get_page_info_by_idx_mrsh.c>
 
-static inline size_t z_vrfy_flash_get_page_count(struct device *dev);
+static inline size_t z_vrfy_flash_get_page_count(struct device *dev)
```

Comment:
Can you pull this out into its own PR, seems like a bug fix that would be nice to get into 2.1 if possible.

Reply:
@galak moved to #21064